doc/qbittorrent.1

```diff
@@ -1,41 +1,269 @@
-.\" Automatically generated by Pandoc 2.9.2
+.\" Automatically generated by Pandoc 1.19.2.4
```

Comment:
Get yourself the latest version here: https://github.com/jgm/pandoc/releases/tag/2.9.2

Reply:
Fixed.